Core/GDCore/IDE/CaptureOptions.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include <memory>
+#include <vector>
+
+#include "GDCore/String.h"
+
+namespace gd {
+
+class GD_CORE_API Screenshot {
+ public:
+  Screenshot();
+  virtual ~Screenshot() {};
+
+  void SetTiming(int timing_) { timing = timing_; }
```

Comment:
I would renamed the filed to be "DelayTimeInSeconds" or "DelayTimeInMs"

so that the unit is clear and "Timing" is a bit vague. Delay Time is I think a tad clearer :)